src/CommitmentsV2/SFA.DAS.CommitmentsV2/Application/Queries/GetCohortSummary/GetCohortSummaryHandler.cs

```diff
@@ -23,6 +23,7 @@ public Task<GetCohortSummaryResponse> Handle(GetCohortSummaryRequest request, Ca
                 .GetById(request.CohortId, c => new GetCohortSummaryResponse
                 {
                     LegalEntityName = c.LegalEntityName,
+                    ProviderName = c.ProviderName,
```

Comment:
We should avoid getting the Provider or Legal Entity names from the Cohort object, and should get them from the corresponding entities directly.

Is it worth deprecating those properties, since they are there only to write to for backwards-compatibility?

Also, is this a good time to rename that Commitment object in the context to Cohorts?